src/python/WMQuality/Emulators/EmulatedUnitTestCase.py

```diff
@@ -21,9 +23,11 @@ class EmulatedUnitTestCase(unittest.TestCase):
     FIXME: For now only DBS is mocked
     """
 
-    def __init__(self, methodName='runTest', mockDBS=True, mockPhEDEx=False):  # FIXME: Default to False for both?
+    def __init__(self, methodName='runTest', mockDBS=True, mockPhEDEx=False,
```

Comment:
This may give you a conflict since we want all of these to be True (and remove the FIXME)